osu.Game.Rulesets.Osu/Difficulty/Preprocessing/OsuDifficultyHitObject.cs

```diff
@@ -128,7 +189,7 @@ private void computeSliderCursorPosition(Slider slider)
 
             slider.LazyEndPosition = slider.StackedPosition;
 
-            float followCircleRadius = (float)(slider.Radius * 2.4);
+            float approxFollowCircleRadius = (float)(slider.Radius * 1.4); // using 1.4 to better follow the real movement of a cursor.
```

Comment:
Still not massively sold on this, understandably. Can you show some difference in values where this is set to `1.4` and `2.4` respectively which justifies this?

Reply:
this was rewritten to use a 1.6 multiplier as a more accurate "real movement" threshold. I consider the logic here to be similar to how circles assume a distance of center to center. even though many overlapping patterns do not actually require such movement.